apps/web/i18n/lang/de.json

```
@@ -222,6 +222,7 @@
       "createAccountLinkLabel": "Jetzt anmelden!",
       "forgotPasswordLabel": "Passwort vergessen?",
       "heading": "In ihr Konto einloggen",
+      "headingSuccess": "Passwort erfolgreich zurücksetzen",
```

```suggestion
      "title": "Passwort zurückgesetzt",
```
Also, this is in the wrong section. Should be in `resetPassword` instead of `auth`.